 URL : "https://github.com/omz/Pythonista-Issues/issues/426" TITLE : no gps access BODY : in pythonista 3.11013 the possibilty to access gps is missing no options in setup available